Docs: Add single weight per day restriction to planned enhancements (#318)
AY2324S2-CS2103T-T17-3/tp: docs/UserGuide.md

| **Note**      | `note INDEX [NOTE]` <br> e.g. <br> `note 2 Sprained right ankle in the past`                                                                                                                        |
| **Weight**    | `WEIGHT INDEX [WEIGHT]` <br> e.g. <br> `weight 3 70`                                                                                                                                                |

## **Appendix: Planned Enhancements**

**1. Restrict the number of weight entries per day**

> Having multiple weight entries within a day distorts the weight-tracking graph while providing no additional value.

Reviewer: ```suggestion
Since the variance of a client's weight values in a day is negligible, we can restrict the number of weight entries per day to 1. Having multiple weight entries within a day also distorts the weight-tracking graph.
```

> Since the variance of a client's weight values in a day is negligible, we can restrict the number of weight entries per day to 1. Having multiple weight entries within a day also distorts the weight-tracking graph.

Reviewer 2: ```suggestion
Since a client's weight won't change much within a day, we can restrict the number of weight entries per day to 1. Having multiple weight entries within a day also distorts the weight-tracking graph.
```
